sdk/appconfiguration/ci.yml

```diff
@@ -34,3 +34,22 @@ extends:
       - name: azure-data-appconfiguration
         groupId: com.azure
         safeName: azuredataappconfiguration
+      - name: azure-spring-cloud-appconfiguration-config
```

Comment:
Maybe we need add more contents. Refs: https://github.com/Azure/azure-sdk-for-java/pull/18916/files#diff-fa1afb9632f48be030514263b2dfff0d3bc6010b5eb620daa034d0e20e46c1b6L13